examples/catset/test.sh

```diff
@@ -38,3 +40,16 @@ kubectl patch $cs nginx-backend --type=merge -p '{"spec":{"replicas":2}}'
 
 echo "Wait for 2 Pods to be Ready..."
 until [[ "$(kubectl get $cs nginx-backend -o 'jsonpath={.status.readyReplicas}')" -eq 2 ]]; do sleep 1; done
+
+echo "Append our own finalizer so we can read the final state..."
+kubectl patch $cs nginx-backend --type=json -p '[{"op":"add","path":"/metadata/finalizers/-","value":"'${finalizer}'"}]'
+
+echo "Delete CatSet..."
+kubectl delete $cs nginx-backend --wait=false
+
+echo "Expect CatSet's finalizer to scale to 0..."
```

Comment:
Expect CatSet's number of replicas to scale to 0?

Reply:
The implicit meaning was intended to be, "Expect CatSet's finalizer to scale the CatSet to 0 replicas". I'll update the comment to add back the elided words for clarity.